doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst

```diff
@@ -65,23 +65,43 @@ There are two implementations currently supported:
   .. figure:: images/bootloader_memory_layout.svg
      :alt: Memory layout
 
-For detailed information about the memory layout, see the partition configuration in the :file:`partitions.yml` file, located in the build folder directory, or run ``ninja partition_manager_report``.
+By default, building an application with any bootloader configuration creates a :ref:`multi-image build <ug_multi_image>`, where the :ref:`partition_manager` manages its memory partitions.
+Bootloaders are built as child images.
+When building an application with :ref:`Cortex-M Security Extensions (CMSE) enabled <app_boards_spe_nspe_cpuapp_ns>`, then :ref:`Trusted Firmware-M (TF-M) <ug_tfm>` is also built with the image automatically.
+From the bootloader perspective, the TF-M is part of the booted application image.
+
+.. _ug_bootloader_flash_static_requirement:
+
+Static partition requirement for DFU
+------------------------------------
+
+By default, the Partition Manager generates the partition map dynamically.
+As long as you are not using Device Firmware Updates (DFU), you can use the dynamic generation of memory partitions.
+
+However, if you want to perform DFU, you must :ref:`define a static partition map <ug_pm_static>`.
+This is important because the dynamically generated partitions change between builds.
+The newly generated application images may no longer be compatible with the bootloader programmed to the device.
```

Comment:
Can I suggest that we specify that it is partitions that are not compatible?
'The newly generated application images may no longer have partitions which are compatible with the bootloader partitions.'
I was not able to phrase this sentence in a way that keeps the ' programmed to the device' part

Reply:
Maybe:
`The newly generated application images may no longer use partition map which is compatible with the partition map used by the bootloader programmed to the device.`?

Reply:
I agree that using "partition map" is good here.
However, keeping the `programmed to the device` at the end makes the sentence quite "heavy". Is it nessecary to specify that the bootloader is already programmed, or can we expect readers to undestand that from context?
Nitpick, add "a": `no longer use a partition map`